configs/sites/tier1/noaa-aws/packages.yaml

```diff
@@ -1,115 +1,114 @@
 packages:
-### Modifications of common packages
-  # Pin flex to avoid duplicate packages
-  flex:
-    version: ['2.6.4']
-### All other external packages listed alphabetically
   bash:
     externals:
-    - spec: bash@4.2.46
+    - spec: bash@4.4.20
       prefix: /usr
   berkeley-db:
     externals:
-    - spec: berkeley-db@5.3.21
+    - spec: berkeley-db@5.3.28
       prefix: /usr
   cpio:
     externals:
-    - spec: cpio@2.11
+    - spec: cpio@2.12
       prefix: /usr
   diffutils:
     externals:
-    - spec: diffutils@3.3
+    - spec: diffutils@3.6
       prefix: /usr
   ecflow::
     buildable: False
     externals:
     - spec: ecflow@5.8.4+ui+static_boost
-      prefix: /contrib/spack-stack/ecflow-5.8.4
+      prefix: /contrib/spack-stack-rocky8/ecflow-5.8.4
+      modules:
+      - ecflow/5.8.4
   file:
     externals:
-    - spec: file@5.11
+    - spec: file@5.33
       prefix: /usr
   findutils:
     externals:
-    - spec: findutils@4.5.11
+    - spec: findutils@4.6.0
       prefix: /usr
-  gawk:
+  flex:
     externals:
-    - spec: gawk@4.0.2
+    - spec: flex@2.6.1
       prefix: /usr
-  gettext:
+  gawk:
     externals:
-    - spec: gettext@0.19.8.1
+    - spec: gawk@4.2.1
       prefix: /usr
   git:
     externals:
-    - spec: git@1.8.3.1~tcltk
+    - spec: git@2.39.3~tcltk
       prefix: /usr
   git-lfs:
     externals:
-    - spec: git-lfs@2.4.1
-      prefix: /contrib/spack-stack/git-lfs-2.4.1
+    - spec: git-lfs@3.2.0
+      prefix: /usr
   gmake:
     externals:
-    - spec: gmake@3.82
+    - spec: gmake@4.2.1
       prefix: /usr
   groff:
     externals:
-    - spec: groff@1.22.2
+    - spec: groff@1.21
       prefix: /usr
   hwloc:
     externals:
-    - spec: hwloc@1.11.8
+    - spec: hwloc@2.2.0
       prefix: /usr
   krb5:
     externals:
-    - spec: krb5@1.15.1
-      prefix: /usr
-  lustre:
-    externals:
-    - spec: lustre@2.12.7
+    - spec: krb5@1.18.2
       prefix: /usr
   mysql:
     externals:
     - spec: mysql@8.0.31
       prefix: /contrib/spack-stack/mysql-8.0.31
+  ncurses:
+    externals:
+    - spec: ncurses@6.1-10.20180224+termlib abi=5
+      prefix: /usr
   openjdk:
     externals:
-    - spec: openjdk@1.8.0_322-b06
+    - spec: openjdk@1.8.0_402-b06
       prefix: /usr
   perl:
     externals:
-    - spec: perl@5.16.3~cpanm+shared+threads
+    - spec: perl@5.26.3~cpanm+shared+threads
       prefix: /usr
   pkg-config:
     externals:
-    - spec: pkg-config@0.27.1
+    - spec: pkg-config@1.4.2
       prefix: /usr
   rsync:
```

Comment:
external qt@5 missing

Reply:
That is now in common/modules.yaml and common/packages.yaml:
```
    qt:
      require: '@5'
```